src/main/java/apprenticeship/Game.java

```diff
@@ -0,0 +1,59 @@
+package apprenticeship;
+
+
+public class Game {
+    private Rules rules = new RulesFor3x3();
+    private Board board = new Board();
+    private HumanPlayer player1;
+    private HumanPlayer player2;
+
+    public Game(HumanPlayer player1, HumanPlayer player2) {
```

Comment:
I like how you injected the players into the `Game`. This will make it easier to add computer players in future stories.